integrationtests/gitjob/controller/controller_test.go

```diff
@@ -217,6 +217,28 @@ var _ = Describe("GitJob controller", func() {
 			}).Should(BeTrue())
 		})
 	})
+
+	When("a new GitRepo is created with DisablePolling set to true", func() {
+		var (
+			gitRepo     v1alpha1.GitRepo
+			gitRepoName string
+		)
+
+		BeforeEach(func() {
+			gitRepoName = "disable-polling"
+		})
+
+		It("should update the commit from the actual repo", func() {
+			gitRepo = createGitRepoWithDisablePolling(gitRepoName)
+			Expect(k8sClient.Create(ctx, &gitRepo)).To(Succeed())
+
+			By("verifying the commit is updated")
+			Eventually(func() string {
+				Expect(k8sClient.Get(ctx, types.NamespacedName{Name: gitRepoName, Namespace: gitRepoNamespace}, &gitRepo)).To(Succeed())
+				return gitRepo.Status.Commit
+			}, "30s", "1s").ShouldNot(Equal(""))
+		})
```

Comment:
I think this test could also check that the job is eventually created, like other tests in this same file.

Reply:
I"ll do it !